docs/contributing/intro.md

```diff
@@ -1,4 +1,4 @@
-# Contributing
+# Introduction
```

Comment:
Note: this guide was taken from a different project and it doesn't currently correspond to the actual code here. For instance, the naming conventions, or even the reference to the "master" branch (our main branch is "main"). We should definitely update the documentation to reflect the real state of the project, but that can come later.